devices.js

```diff
@@ -195,6 +195,27 @@ const preset = {
             toZigbee: preset.light_onoff_brightness_colortemp_color({supportsHS: true, ...options}).toZigbee.concat([tz.ledvance_commands]),
         }),
     };
+    preset.tradfri = {
+        light_onoff_brightness: (options = {}) => ({
+            ...preset.light_onoff_brightness(options),
+            exposes: preset.light_onoff_brightness(options).exposes.concat([
+                exposes.enum('power_on_behavior', ea.ALL, ['off', 'previous', 'on'])
+                    .withDescription('Controls the behaviour when the device is powered on')]),
```

Comment:
To prevent this from duplicated 3 times, add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/2443f127b9051e6e83035faa5aa4ae5afa0bf3d0/lib/exposes.js#L510 `power_on_behavior : new () => Enum('power_on_behavior', ea.ALL, ['off', 'previous', 'on']).withDescription('Controls the behaviour when the device is powered on')` and use `e. power_on_behavior()` here.